README.md

```diff
@@ -285,27 +285,26 @@ _The agent calls `getDresses(6)` and receives a JSON object:_
 
 ```json
 {
-    products: [
+    "products": [
         {
-            id: 1021,
-            description: "A short sleeve long dress with full length button placket...",
-            price: "€180",
-            image: "img_1024.png",
+            "id": 1021,
+            "description": "A short sleeve long dress with full length button placket...",
+            "price": "€180",
+            "image": "img_1024.png",
         },
         {
-            id: 4320,
-            description: "A straight midi dress in organic cotton...",
-            price: "€140",
-            image: "img_4320.png",
+            "id": 4320,
+            "description": "A straight midi dress in organic cotton...",
+            "price": "€140",
+            "image": "img_4320.png",
         },
         ...
     ]
 }
 ```
 
 > [!Note]
-> How to pass images and other non-textual data is something we should improve 
-> Issue #10
+> How to pass images and other non-textual data is something we should improve (See [Issue #10](https://github.com/webmachinelearning/webmcp/issues/10))
```

Comment:
Double checking: is this the correct issue?

There's also https://github.com/webmachinelearning/webmcp/issues/41

Reply:
I stand corrected. #41 seems like a better fit.

Reply:
Ok, updated